test/functional/apps/discover/group3/_lens_vis.ts

```diff
@@ -574,7 +575,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await header.waitUntilLoadingHasFinished();
       await discover.waitUntilSearchingHasFinished();
 
-      expect(await getCurrentVisTitle()).to.be('Bar');
+      // type stays the same as the columns remain the same
+      expect(await getCurrentVisTitle()).to.be('Treemap');
```

Comment:
```suggestion
      expect(await getCurrentVisTitle()).to.be('Bar');
```

What query would reset the chart type back to the default? Can we have a test for this case here instead?

Reply:
Going back to a histogram, I will yes 👍